### PR TITLE
Add preferBemjson option

### PR DESCRIPTION
**Pull request by skad0**
As requested in #8 
@zxqfox @blond

### CODE REVIEW

**Comment by skad0** [comment]
I have some doubts about should I throw an exception, when neither bemjson/bemdecl given? Or this is about BemBundle?

**Comment by qfox** [comment]
I'd say we should skip a directory without needed files. No need to throw

File: index.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ module.exports = function(pattern, opts) {
                     const decl = res[0];
                     const bemjson = res[1];
 
-                    decl && (bundle.decl = decl);
+                    (decl && !opts.preferBemjson) && (bundle.decl = decl);
```
Comment (tadatuta) [inline index.js]: скобки не нужны

**Comment by tadatuta** [comment]
LGTM

**Pull request merged**